src/base/net/downloadhandler.cpp

```diff
@@ -205,58 +204,53 @@ void Net::DownloadHandler::handleRedirection(QUrl newUrl)
 
 namespace
 {
-    QString tr(const char *message)
-    {
-        return QCoreApplication::translate("DownloadHandler", message);
-    }
-
     QString errorCodeToString(QNetworkReply::NetworkError status)
     {
         switch (status) {
         case QNetworkReply::HostNotFoundError:
-            return tr("The remote host name was not found (invalid hostname)");
+            return QCoreApplication::translate("DownloadHandler", "The remote host name was not found (invalid hostname)");
```

Comment:
At least use named constant for context name.

Reply:
No, that won't work. The i18n string doesn't show up in .ts file.

Reply:
The another way is to make `errorCodeToString()` static in DownloadHandler class. But I can't say I like it better.

Reply:
You're free to change it or leave as is. I approve it both way.